### ISSUE TITLE
File modes should always be encapsulated in quotation marks

### ISSUE THREAD

**Issue by jtprogru** [warpnet/salt-lint#248]
`salt-lint` don't check close quote in DOUBLE quote in `mode: '0700`

Example `.sls` file `cat example.sls`:

```yaml
{%- if HOST_IS_BACKUP_SERVER == True %}

{#- create custom PGDATA_DIR @backup server #}
{{ PGDATA_DIR }}:
  file.directory:
    - user: postgres
    - mode: '0700
    - makedirs: True
    - require:
      - pkg: postgresql{{ PGSQL_VERSION }}-pkg

{#- create custom /etc/sysconfig/ config @backup #}
/etc/sysconfig/pgsql/{{ PATRONI_SERVICE_NAME }}:
  file.managed:
    - contents: |
        # salt: {{ STATE_NAME }}
        #PGDATA={{ PGDATA_DIR }}
        PGPORT={{ PGSQL_PORT }}
        BACKUP_METHOD=pg_dump
    - user: root
    - mode: "0644
    - require:
      - file: {{ PATRONI_CONFIGURATION }}
      - service: {{ PATRONI_SERVICE_NAME }}-service

{%- endif %}
```

Run:
```bash
salt-lint example.sls
Examining example.sls of type sls
```

Environment:
```bash
salt-lint --version
salt-lint 0.5.2
```

OS: this reproduced on macOS, CentOS, Ubuntu

**Comment by jtprogru** [comment]
Any ideas? `yamllint`  does not catch this error.

**Comment by roaldnefs** [comment]
Thanks @jtprogru for your bug report!

I hope to find some time to fix some bugs in the near future.
